libcontainer/container_linux.go

```diff
@@ -291,6 +302,37 @@ func (c *linuxContainer) Signal(s os.Signal, all bool) error {
 	return nil
 }
 
+func (c *linuxContainer) createExecFifo() error {
+	rootuid, err := c.Config().HostUID()
+	if err != nil {
+		return err
+	}
+	rootgid, err := c.Config().HostGID()
+	if err != nil {
+		return err
+	}
+
+	fifoName := filepath.Join(c.root, execFifoFilename)
+	if _, err := os.Stat(fifoName); err == nil {
```

Comment:
I don't think so, `syscall.Mkfifo` will call `mknodat` syscall, and according to https://linux.die.net/man/2/mknodat ,there is no EEXIST error.